python/examples/1.wdl

```diff
@@ -14,7 +14,7 @@ task grep {
 
 task wc {
   command {
-    wc -l ${sep=' ' File files+} | tail -1 | cut -d' ' -f 2
+    wc -l ${sep=' ' File files+} | tail -1 | tr -s ' ' | cut -d' ' -f 2
```

Comment:
I suspect this is because I'm using the GNU versions of all of these tools and the default on Mac OS X is BSD-style versions of all of these tools.  Fine 95% of the time until you notice whitespace differences that and odd quirks that differ between them.

Surprisingly this is not the first time this has happened this week.